### PR TITLE
update to proper message on a successful fence op

### PR DESCRIPTION
**Pull request by rakeshgm**

### CODE REVIEW

**Review by yati1998 — APPROVED** [review]
LGTM

**Review by yati1998 — CHANGES_REQUESTED** [review]
@rakeshgm please check the ci failure once.

**Comment by rakeshgm** [comment]
Not sure, what the CI error means here. I don't have any uncommitted changes.

File: apis/csiaddons/v1alpha1/networkfence_types.go [hunk 1]
```diff
@@ -40,6 +40,14 @@ const (
 	FencingOperationResultFailed FencingOperationResult = "Failed"
 )
 
+const (
```
Comment (Madhu-1) [inline apis/csiaddons/v1alpha1/networkfence_types.go:43]:
```
[🎩︎]mrajanna@li-2cfbef4c-22d9-11b2-a85c-a3e4a93c405f kubernetes-csi-addons $]go fmt ./...
apis/csiaddons/v1alpha1/networkfence_types.go
[🎩︎]mrajanna@li-2cfbef4c-22d9-11b2-a85c-a3e4a93c405f kubernetes-csi-addons $]git diff
diff --git a/apis/csiaddons/v1alpha1/networkfence_types.go b/apis/csiaddons/v1alpha1/networkfence_types.go
index 2a532e92..71b891c3 100644
--- a/apis/csiaddons/v1alpha1/networkfence_types.go
+++ b/apis/csiaddons/v1alpha1/networkfence_types.go
@@ -41,10 +41,10 @@ const (
 )
 
 const (
-       // FenceOperationSuccessfulMessage represents successful message on fence operation 
+       // FenceOperationSuccessfulMessage represents successful message on fence operation
        FenceOperationSuccessfulMessage = "fencing operation successful"
-       
-       // UnFenceOperationSuccessfulMessage represents successful message on unfence operation 
+
+       // UnFenceOperationSuccessfulMessage represents successful message on unfence operation
        UnFenceOperationSuccessfulMessage = "unfencing operation successful"
 )
```

Comment (rakeshgm) [reply apis/csiaddons/v1alpha1/networkfence_types.go:43]: ok, I see, go fmt changed the lines and hence the uncommitted changes, I have updated the patch

**Review by Madhu-1 — APPROVED** [review]

**Review by riya-singhal31 — APPROVED** [review]

**Comment by nixpanic** [comment]
@mergifyio refresh

**Pull request merged**